client/src/Components/Contact.js

```diff
@@ -0,0 +1,44 @@
+import React, { Component } from 'react'
+import { Link } from '@reach/router'
+import '../Style/Contact.css'
+
+export default function Contact(props) {
+    
+    return (
+        <div id = "contact-body">
+       <div id= "contact-head">
+           Let's Make Things Happen...
+       
+<br></br>
+<br></br>
+<a href = 'https://www.jpdgallery.com/' target="_blank">jpdgallery.com</a>
+<br></br>
+<br></br>
+
+
+<a href = 'https://www.linkedin.com/in/jena-durell-b5044b89/' target="_blank">Linkedin</a>
+<br></br>
+<br></br>
+<a href = 'https://www.instagram.com/snowy_vt/?hl=en' target="_blank">Instagram</a>
+<br></br>
+<br></br>
+<br></br>
+<br></br>
+<br></br>
+<br></br>
+<br></br>
+<br></br>
+<br></br>
```

Comment:
`<br>` tags shouldn't be used for styling! This adds bloat and makes your code less readable. This same thing can be accomplished with 1 line of css